catalog/app/containers/Admin/Members.js

```diff
@@ -0,0 +1,134 @@
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+import PT from 'prop-types';
+import React, { Fragment } from 'react';
+import { FormattedMessage as FM, injectIntl } from 'react-intl';
+import { Link } from 'react-router';
+import { compose, setPropTypes, setDisplayName } from 'recompose';
+
+import Spinner from 'components/Spinner';
+import api, { apiStatus } from 'constants/api';
+
+import msg from './messages';
+import { branch, formatActivity, formatDate, withStatefulActions } from './util';
+import ErrorMessage from './ErrorMessage';
+import Cell from './LockableCell';
+import SettingsMenu from './SettingsMenu';
+
+
+const memberActivities = [
```

Comment:
logically, i'm not sure what this list corresponds to; it is neither columns in a table nor a list of activity types.

Reply:
these are all the fields returned by the api